exporter/containerimage/writer.go

```diff
@@ -214,10 +239,74 @@ func (ic *ImageWriter) exportLayers(ctx context.Context, refCfg cacheconfig.RefC
 	return out, err
 }
 
+func (ic *ImageWriter) exportAttestations(ctx context.Context, s session.Group, desc *ocispecs.Descriptor, attestations ...exporter.Attestation) ([]intoto.Statement, error) {
+	span, ctx := tracing.StartSpan(ctx, "export attestations")
+
+	eg, ctx := errgroup.WithContext(ctx)
+	attestationsDone := oneOffProgress(ctx, "exporting attestations")
```

Comment:
I see 3 different calls to `oneOffProgress` for what looks to be the same action for the user. Is this separation really needed.